source/mainnet/net/web3-id/wallet.rst

```diff
@@ -0,0 +1,139 @@
+- Revoked: the verifiable credential is no longer valid because the holder or issuer has canceled it
+- Not activated: the verifiable credential is issued but is not yet active because its *valid from date* has not passed, e.g. a concert ticket
+- Expired: the verifiable credential is past the *valid until date*, e.g. a concert ticket
+
+Add verifiable credentials to the wallet
+========================================
+
+When choosing to add verifiable credentials from an issuer, make sure that you *trust* the issuer.
+
+#. Contact the issuer of the verifiable credentials using the dApp the issuer provides and provide the information requested by the issuer.
+
+#. The issuer sends a request to your |bw| to issue a verifiable credential. Click **Cancel** to stop the process or click **Add credential** to continue. You will see the verifiable credential as **Pending**.
+
+    .. image:: ../images/browser-wallet/add-web3id-credential.png
+        :alt: window with pending credential and option buttons
+        :width: 50%
+
+To see the details of the verifiable credential:
+
+#. Click |hamburger-bw| and select **Verifiable credentials**.
+
+#. Click on the verifiable credential card.
+
+#. Click |actions-bw| and select **Details**.
+
+    .. image:: ../images/browser-wallet/vc-details.png
+        :alt: window with card showing details of the verifiable credential
+        :width: 50%
+
+Backup verifiable credentials
+=============================
+
+#. Click |hamburger-bw| and select **Verifiable credentials**.
+
+#. Click |actions-bw| and select **Download export file**. This saves the verifiable credentials in a file called *web3IdCredentials.export* in your downloads folder by default.
+
+Revoke verifiable credentials
+=============================
+
+In some cases, the issuer of the verifiable credential may allow you to revoke a credential yourself in your |bw| if you no longer need it.
+
+#. Click |hamburger-bw| and select **Verifiable credentials**.
+
+#. Click on the verifiable credential card to revoke.
+
+#. Click |actions-bw| and select **Revoke**.
+
+#. Review the transaction information and click **Send**.
+
+Recover verifiable credentials
+==============================
+
+#. Click |hamburger-bw| and select **Verifiable credentials**.
+
+#. Click |actions-bw| and select **Open import window**.
+
+#. Click **Select file to import**. Navigate to the location that the *web3IdCredentials.export* file is located and select the file.
+
+    .. image:: ../images/browser-wallet/vc-import.png
+        :alt: window with button to navigate to import file location
+        :width: 50%
+
+#. The verifiable credentials that could be imported from the file are added to the wallet.
+
+Use verifiable credentials
+==========================
+
+In cases where a verifier asks to prove your verifiable credentials, you are asked to prove or reveal information in the |bw|. Proofs or revealing information can be:
```

Comment:
"asks to prove your verifiable credentials" seems incorrect.

Have we at this point introduced the "presentation" terminology?